index.ts

```diff
@@ -69,7 +70,7 @@ function checkAuth(req, res, next) {
 }
 
 if (!oauthEnabled) {
-    app.get("/", function (req, res) {
+    app.get("/", function (req: express.Request, res: express.Response) {
```

Comment:
for better readability:
~~~typescript
import {Request, Response} from 'express';
// ...
app.get("/", function (req: Request, res: Response) { //...
~~~